app/controllers/setup_controller.rb

```diff
@@ -93,4 +94,15 @@ def minion_not_found(exception)
       format.json { render json: exception.message, status: :not_found }
     end
   end
+
+  def check_empty_settings
+    return unless settings_params.values.any?(&:empty?)
```

Comment:
Not sure if it makes sense for such a simple case but if we need more validations like this one, we could introduce Form objects (e.g. [point 3 here](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)).